File: code/fe/src/types/location.ts
export enum LocationMode {
  ONLINE = "ONLINE",
  CAMPUS_1 = "CAMPUS_1",
  CAMPUS_2 = "CAMPUS_2",
}

export const LocationModeLabels: Record<LocationMode, string> = {
  [LocationMode.ONLINE]: "Online (Meet/Zoom)",
  [LocationMode.CAMPUS_1]: "Campus 1 (Lý Thường Kiệt)",
  [LocationMode.CAMPUS_2]: "Campus 2 (Dĩ An)",
};

File: code/be/app/models/enums/location.py
from enum import Enum


class LocationMode(str, Enum):
    """Location mode for tutoring sessions and availability slots."""
    ONLINE = "ONLINE"  # Online session -> location is meeting link
    CAMPUS_1 = "CAMPUS_1"  # Campus 1 - Lý Thường Kiệt -> location is room in c1
    CAMPUS_2 = "CAMPUS_2"  # Campus 2 - Dĩ An -> location is room in c2
